Gemfile.lock

```diff
@@ -0,0 +1,158 @@
+GIT
```

Comment:
Yes, absolutely. The lock file is essential to the whole process of using Bundler. Only when you check in the lock file you guarantee repeatable builds/setups. There is a argument for not checkin it in for library gems, but imo it is still better to have it checked in and then do dependencies upgrades at dedicated points in time.